src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -121,29 +155,34 @@ public async Task StopAsync(CancellationToken cancellationToken = default)
         await _host.StopAsync(cancellationToken).ConfigureAwait(false);
     }
 
-    private void SuppressLifetimeLogsDuringManifestPublishing()
+    private void SuppressLogs()
     {
         var config = (IConfigurationRoot)_host.Services.GetRequiredService<IConfiguration>();
-        var options = _host.Services.GetRequiredService<IOptions<PublishingOptions>>();
-
-        if (options.Value?.Publisher != "manifest")
-        {
-            // If we aren't doing manifest publishing we want the logs
-            // to be produced as normal.
-            return;
-        }
 
+        // Suppress excess logging from lifetime logger category. This blocks the "Now listening on"
+        // message. This is not a problem for VS usage, but it does confuse "dotnet watch" which looks
+        // at the stdout to figure out when it should launch the browser. Rather than relying on
+        // lifetime logger category for this special string we will emit it ourselves to make
+        // sure that it is correct.
         var hostingLifetimeLoggingLevelSection = config.GetSection("Logging:LogLevel:Microsoft.Hosting.Lifetime");
         hostingLifetimeLoggingLevelSection.Value = "Warning";
 
+        // Suppresses warning that we've overridden the default ASPNETCORE_URLS value. This is
+        // because the endpoints that we host in the AppHost is a randomly assinged resource
+        // endpoint that is used for the dashboard. The ASPNETCORE_URLS value is actually passed
+        // into the dashboard that is launched out of process.
+        var aspNetCoreServerKestrelLoggingLevelSection = config.GetSection("Logging:LogLevel:Microsoft.AspNetCore.Server.Kestrel");
+        aspNetCoreServerKestrelLoggingLevelSection.Value = "None";
+
```

Comment:
We should do this in the resource server logic instead of the above.

https://github.com/dotnet/aspire/blob/7e4a7cc701bf8291ca606cefcdf860ea4eebad53/src/Aspire.Dashboard/DashboardWebApplication.cs#L29-L30

Reply:
Should be able to do that.

Reply:
Done.